### DISCUSSION TITLE
this script has a problem

### DISCUSSION THREAD

**Discussion by viponedream** [swar/Swar-Chia-Plot-Manager#345] [Q&A]
i have  three jobs
- name: A
- name: B
- name: C

if i run this script, these 3 job will run at the same time immediately.  you can figure it out, it will finish at the same time too.
it cause the problem that  they will write to the  destination_directory at the same time too.   that will be very slow.

it need a global var better to set it delay.   

delay   0 :  start all jobs immediately
delay 10:   start next job after 10 minutes.

**Comment by Arigasendo** [comment]
Swar merged a branched called initialdelay into the development branch, which adds this job config:

initial_delay_minutes - This is the initial delay that is used when initiate the first job. It is only ever considered once. If you restart manager, it will still adhere to this value.

I haven't checked if the development branch currently works but you could try it out.

**Comment by swar** [comment]
This really isn't a problem. It's what it was designed to do. Please use Github Discussions next time unless there's an actual issue with the software.
